tests/perf/prerequisites/event-bus/setup.sh

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env bash
```

Comment:
Just curious: is there any reason for substituting the Go client with a Bash script? Over time fragmented testing approaches can cause a great deal of headaches.